docs/source/dispatcher/filters.rst

```diff
@@ -111,6 +111,14 @@ ExceptionsFilter
     :show-inheritance:
 
 
+ExceptionsFilter
```

Comment:
Change to `IdFilter`

Reply:
Done with [80b1168](https://github.com/aiogram/aiogram/pull/172/commits/80b11684801446a4e457214738adde4be1018ccd)